index.html

```diff
@@ -1131,14 +1134,14 @@ <h4>
             "RemotePlaybackState">connected</a> through an icon or other means.
           </p>
           <div class="note">
-            The mechanism that is used to connect the user agent with the
-            <a>remote playback device</a> and play the <a>remote playback
-            source</a> is an implementation choice of the user agent. The
-            connection will likely have to provide a two-way messaging
-            abstraction capable of carrying media commands to the remote
-            playback device and receiving media playback state in order to keep
-            the <a>media element state</a> and <a>remote playback state</a> in
-            sync (unless <a>media mirroring</a> is used).
+            The mechanisms that are used to connect the user agent with the
+            <a>remote playback device</a>, select the <a>remote playback
+            source</a> and initiate playback are implementation choices of the
+            user agent. The connection will likely have to provide a two-way
```

Comment:
Nit: The connection provides a two way messaging abstraction? Isn't that the connection itself, and the user agent and remote playback device have to provide the messaging abstractoin?

Reply:
I think the "connection"  refers in an abstract way with the communication mechanism, which can be used for syncing media element state or other purposes.  As this was pre-existing text in the spec, I'm inclined to leave it alone in this PR.